src/pages/plugin/plugin.js

```diff
@@ -231,6 +234,8 @@ export default async function PluginInclude(
 				$button.textContent = oldText;
 			}
 		} catch (error) {
+			window.log("error", "Failed to buy:");
```

Comment:
@bajrangCoder I think you forgot to append the caught `error` to the log message

Reply:
No , check second line.
It's like that because if I will pass `Error` object combined with the the string then the `Error` object will become string and then it will not show the full error